pkgs/by-name/co/cosmic-initial-setup/package.nix

```diff
@@ -0,0 +1,86 @@
+{
+  lib,
+  stdenv,
+  rustPlatform,
+  fetchFromGitHub,
+  just,
+  libcosmicAppHook,
+  libinput,
+  openssl,
+  udev,
+  nixosTests,
+  nix-update-script,
+}:
+rustPlatform.buildRustPackage {
+  pname = "cosmic-initial-setup";
+  version = "0-unstable-2025-09-04";
+
+  src = fetchFromGitHub {
+    owner = "pop-os";
+    repo = "cosmic-initial-setup";
+    rev = "fb6b147a847cbfc732b292034fd5f91e6246ddc2";
+    hash = "sha256-bw6WOha3RCc4iNANId3YkNPMpGXxpWZPb/cQf6foAPE=";
+  };
+
+  cargoHash = "sha256-Q0shfFWRXc7zeHMP+Ruy5u3hW6daP3e7eGOz2OcUhxU=";
+
+  # cargo-auditable fails during the build when compiling the `crabtime::function`
+  # procedural macro. It panics because the `--out-dir` flag is not passed to
+  # the rustc wrapper.
+  auditable = false;
+
+  nativeBuildInputs = [
+    libcosmicAppHook
+    just
+  ];
+
+  buildInputs = [
+    libinput
+    openssl
+    udev
+  ];
+
+  postPatch = ''
+    substituteInPlace justfile \
```

Comment:
Our current contributing guidelines encourage you to also explain why you used `substituteInPlace`:

>   - If any (especially opinionated) patch or `substituteInPlace` is applied, document why.